kaggle_environments/schemas.json

```diff
@@ -109,6 +109,16 @@
       "observation": {
         "description": "Observation to create an action based upon.",
         "type": "object",
+        "additionalProperties": true,
+        "properties": {
+          "remainingOverageTime": {
+            "description": "Remaining per-step timeout overage -- agent is disqualified by TIMEOUT when this drops below 0.",
```

Comment:
nit: `per-step` doesn't feel right since the overage is for all steps?

Reply:
Fair point, I meant to say here that this time is in addition to your per-step `actTimeout`, but I agree it's confusing as-is.